peps/pep-0770.rst

```diff
@@ -141,6 +146,21 @@ In addition to the above, an informational PEP will be created for tools
 consuming included SBOM documents and other Python package metadata to
 generate complete SBOM documents for Python packages.
 
+Terminology
+-----------
+
+This section describes terminology used later in the document:
+
+* **Root SBOM directory**: This is the directory within a Python project source
```

Comment:
ah, this is similar to how PEP-639 does it with `root license directory`
https://github.com/python/peps/blob/9b8e3f9be205b1ea9bb787145c61b5791091f97c/peps/pep-0639.rst?plain=1#L236-L251

maybe you could copy/adapt their style. several years of grind, and multiple readings made this PEP-639 my personal favorite regarding understandability/readability.

Reply:
Ah great idea, I'll do that.

Reply:
Done in https://github.com/python/peps/pull/4222/commits/5a2bea568148a6d4bc8d8821024b6f0ad1400c00